content/docs/fields/date.md

```diff
@@ -13,47 +13,55 @@ The `date` field represents a date and time picker. It can be used for data valu
+| `dateFormat`  | Defines the format for the date. It accepts any [Moment.js date format](https://momentjs.com/docs/#/displaying/format/) (not in localized format). If true the date will be displayed using the defaults for the current locale. See [react-datetime docs](https://github.com/YouCanBookMe/react-datetime) for more details. _(Optional)_                                                                                 |
+| `timeFormat`  | Defines the format for the time. It accepts any [Moment.js time format](https://momentjs.com/docs/#/displaying/format/) (not in localized format). If true the time will be displayed using the defaults for the current locale. If false the timepicker is disabled and the component can be used as datepicker. See [react-datetime docs](https://github.com/YouCanBookMe/react-datetime) for more details._(Optional)_ |
 
-- `name`: The path to some value in the data being edited.
-- `component`: The name of the React component that should be used to edit this field. Available field component types are [defined here](/docs/fields)
-- `label`: A human readable label for the field. This label displays in the sidebar and is optional. If no label is provided, the sidebar will default to the name.
-- `description`: An optional description that expands on the purpose of the field or prompts a specific action.
+> ### FieldConfig
+>
+> This interfaces only shows the keys unique to the date field.
+>
+> Visit the [Field Config](/docs/fields) docs for a complete list of options.
+>
+> ### DatetimepickerProps
+>
+> Any extra properties added to the the `date` field definition will be passed along to the [`react-datettime`](https://www.npmjs.com/package/react-datetime) component. [Moment.js](https://momentjs.com/docs/#/displaying/format/) format is used. See the full list of options [here](https://www.npmjs.com/package/react-datetime#api).
 
-#### Additional Options
+## Example
 
-Any extra properties added to the the `date` field definition will be passed along to the [`react-datettime`](https://www.npmjs.com/package/react-datetime) component. [Moment.js](https://momentjs.com/docs/#/displaying/format/) format is used. See the full list of options [here](https://www.npmjs.com/package/react-datetime#api).
+Below is an example of how a `date` field could be used to edit a `created_at` date on a blog post.
+
+```javascript
+const BlogPostForm = {
+  fields: [
+    {
+      label: 'Date',
+      name: 'created_at',
+      component: 'date',
+      dateFormat: 'MMMM DD YYYY',
+      timeFormat: false,
+    },
+    // ...
+  ],
+}
+```
```

Comment:
Would it be worth it also linking to a real-world example like [BlogPostCreator]( https://github.com/tinacms/tinacms.org/blob/e217a45a2408ff6fd713939885f886a342953a40/tinacms/BlogPostCreator.ts#L19)  or [blog slug page](https://github.com/tinacms/tinacms.org/blob/e217a45a2408ff6fd713939885f886a342953a40/pages/blog/%5Bslug%5D.tsx#L165) (tina cms site).  That way folks could see a working example. or would this just cause problems for maintenance?

Reply:
Hmm 🤔 These docs are meant to be purely Tina (i.e. no reference to frameworks or backends) but I definitely see the value in that. 

What do the other@tinacms/maintainers think?

Reply:
Shit my mention didn't work.

@tinacms/maintainers

Reply:
Yeah I'd rather just have the examples in this doc. If there are other use cases that should be noted, let's add them!